test/jdk/java/lang/RuntimeTests/loadLibrary/exeLibraryCache/LibraryFromCache.java

```diff
@@ -42,6 +42,8 @@
 
 public class LibraryFromCache {
     public static void main(String[] args) throws IOException {
+        System.out.println("os.version = " + System.getProperty("os.version"));
```

Comment:
The copyright end year needs to be updated.

Reply:
Thanks for catching.